nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -592,9 +592,8 @@ sub multiLineList {
 my $networkingDhcpConfig = generateNetworkingDhcpConfig();
 
 my $hwConfig = <<EOF;
-# Do not modify this file!  It was generated by ‘nixos-generate-config’
-# and may be overwritten by future invocations.  Please make changes
-# to /etc/nixos/configuration.nix instead.
+# Automatically generated from 'nixos-generate-config', future invocations
+# may overwrite /etc/nixos/hardware-configuration.nix.
```

Comment:
rather than directly stating the path here, maybe replace this with "future invocations may overwrite the contents of this file"

Reply:
good idea!